addon/tailwind/components/bourbon-modal.css

```diff
@@ -11,20 +11,20 @@
   @apply .bourbon-pin-l;
 }
 
-.bourbon-modal__fixed {
+.BourbonModal-overlay {
+  position: fixed;
+  @apply .bourbon-bg-shade;
+}
+
+.BourbonModal--fixed {
```

Comment:
This rule should probably be below the `.BourbonModal` rule. Or renamed if that wouldn't make sense.